module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -2259,13 +2266,16 @@ public function cancelILLRequests($cancelDetails)
     /**
      * Get Cancel ILL Request Details
      *
-     * @param array $details An array of item data
+     * @param array $request An array of request data
+     * @param array $patron  Patron information from patronLogin
      *
      * @return string Data for use in a form field
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function getCancelILLRequestDetails($details)
+    public function getCancelILLRequestDetails($request, $patron)
```

Comment:
Same question here: should we add a default $patron value to all of these too?

Reply:
Same answer.. :)

Reply:
...and same comment: there are some drivers that have not had the $patron parameter added in this PR (possibly just PAIA and VoyagerRestful in this case).

Reply:
...and same action: commit f6d1968707c94103d8c8ce61b78f26a282cf2151